test/functional/api/cas/ioclass_config.py

```diff
@@ -21,22 +21,25 @@
 
 MAX_IO_CLASS_ID = 32
 MAX_IO_CLASS_PRIORITY = 255
+DEFAULT_IO_CLASS_ID = 0
+DEFAULT_IO_CLASS_PRIORITY = 255
+DEFAULT_IO_CLASS_RULE = "unclassified"
 MAX_CLASSIFICATION_DELAY = timedelta(seconds=6)
 IO_CLASS_CONFIG_HEADER = "IO class id,IO class name,Eviction priority,Allocation"
 
 
 @functools.total_ordering
 class IoClass:
     def __init__(self, class_id: int, rule: str = '', priority: int = None,
-                 allocation: bool = True):
+                 allocation: str = "1.00"):
         self.id = class_id
         self.rule = rule
         self.priority = priority
         self.allocation = allocation
 
     def __str__(self):
         return (f'{self.id},{self.rule},{"" if self.priority is None else self.priority}'
-                f',{int(self.allocation)}')
+                f',{self.allocation}')
 
     def __eq__(self, other):
```

Comment:
not sure what is the purpose of __eq__, but maybe it would make sense to compare self.allocation and other.allocation as a number rather than string, so that "1" == "1.0". For your consideration

Reply:
In case of negative testing it may be valuable to have possibility to use a characters other than numeric